src/Sylius/Abstraction/StateMachine/src/CompositeStateMachine.php

```diff
@@ -40,34 +41,40 @@ public function __construct(
     }
 
     /**
-     * @throws \Exception
+     * @throws StateMachineExecutionException
```

Comment:
What do you think about short syntax?

```suggestion
    /** @throws StateMachineExecutionException */
```